tests/rbd_mirror/test_rbd_mirror_journal_to_snap.py

```diff
@@ -46,6 +75,10 @@ def run(**kw):
             **kw,
         )
 
+        mirrormode = mirror1.get_mirror_mode(imagespec)
+        # Verification of Negative test as image should not allow write operation from secondary
+        test_write_from_secondary(mirror2, mirrormode, imagespec)
```

Comment:
The return value from this method should be returned back to run() method. So I guess you need to write something like

if test_write_from_secondary(mirror2, mirrormode, imagespec):
   return 1

Reply:
Done